docs/src/Contributing.md

```diff
@@ -28,9 +28,3 @@ Currently a number of checks are run per commit for a given PR.
   are consistent and generate valid output.
 - `Tests` runs the file `test/runtests.jl`,  using `Pkg.test()`. These are a mix of
   unit tests and fast integration tests.
-
-We use [`bors`](https://bors.tech/) to manage merging PR's in the the `ClimaLSM` repo.
```

Comment:
could you also delete `docs/src/folder_structure.md` line 21? then we should finally be rid of bors